scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/SparkeyIO.scala

```diff
@@ -33,7 +33,7 @@ import scala.collection.mutable
 import scala.jdk.CollectionConverters._
 
 /** Special version of [[com.spotify.scio.io.ScioIO]] for use with sparkey methods. */
-case class SparkeyTestIO[T] private (path: String) extends TestIO[T] {
+private[sparkey] case class SparkeyTestIO[T](path: String) extends TestIO[T] {
```

Comment:
It seemed confusing to support both this constructor as well as SparkeyIO#apply, [below](https://github.com/spotify/scio/blob/v0.14.0/scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/SparkeyIO.scala#L48).